drivers/hmis/app/models/hmis/filter/enrollment_filter.rb

```diff
@@ -49,4 +50,35 @@ def with_project_types(scope)
   def with_search_term(scope)
     with_filter(scope, :search_term) { scope.matching_search_term(input.search_term) }
   end
+
+  def with_household_tasks(scope)
+    with_filter(scope, :household_tasks) do
+      if input.household_tasks&.include?('ANNUAL_DUE')
+        entry_anniversary = Arel.sql('make_date(extract(year from current_date)::integer, extract(month from "EntryDate")::integer, extract(day from "EntryDate")::integer)')
+        start_date = Arel.sql('make_date(extract(year from current_date)::integer, extract(month from "EntryDate")::integer, extract(day from "EntryDate")::integer) - interval \'30 days\'')
```

Comment:
This is fine with me.  We could probably re-write it using the `nf` method in ArelHelper, but I don't think it'd be any more clear and it wouldn't be any more portable.  I save leave it as is.